api/src/mutations/mint.rs

```diff
@@ -404,6 +404,7 @@ impl Mutation {
             created_by: Set(user_id),
             compressed: Set(compressed),
             credits_deduction_id: Set(Some(credits_deduction_id)),
+            edition: Set(-1),
```

Comment:
Thought the default was already -1.

Reply:
confirmed.

Reply:
yeah sorry I had to remove this and merge it but I forgot